google/cloud/storage/_opentelemetry_meter.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2014 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Manages OpenTelemetry metrics creation and handling"""
+
+try:
+    from opentelemetry import metrics
+    from opentelemetry.sdk.metrics import Counter, MeterProvider
```

Comment:
Only a theoretical concern right now since the built in SDK is the _only_ SDK, but in general libraries like this should only depend on the API package so users can bring their own non-standard SDK if they want.

Reply:
Discussed offline. For now we will actually need the sdk to be imported.